_data/resources/videos.yml

```
@@ -1,3 +1,13 @@
+- title: "Following LINKS Automatically with Scrapy CrawlSpider"
+  url: https://www.youtube.com/watch?v=o1g8prnkuiQ&t=423s
```

Why not start at the beginning of the vídeo?

```suggestion
  url: https://www.youtube.com/watch?v=o1g8prnkuiQ
```